src/org/etools/j1939_84/bus/j1939/packets/LampStatus.java

```diff
@@ -56,9 +57,54 @@ public static LampStatus getStatus(int onOff, int flash) {
         return LampStatus.OTHER;
     }
 
+    /*
+        FOR TESTING PURPOSES ONLY!
+        Can only be used with the values encoded as follows:
+            ALTERNATE_OFF {0, 0}
+            FAST_FLASH {1, 1}
+            NOT_SUPPORTED {3, 3}
+            OFF {0, 3}
+            ON {1, 0}
+            OTHER {3, 2}
+            SLOW_FLASH {1, 3}
+
+        Any packets encoded with any of the other values will
+        fail comparisions as the additional values can not be
+        created.
+     */
+    public static int[] getBytes(LampStatus lampStatus) {
+        int[] bytes; // lampStatus bytes {On/Off, Flash/State}
+        //
+        switch (lampStatus) {
+        case ALTERNATE_OFF:
+            bytes = new int[] { 0, 0 };
+            break;
+        case FAST_FLASH:
+            bytes = new int[] { 1, 1 };
+            break;
+        case NOT_SUPPORTED:
+            bytes = CollectionUtils.join(new int[] { 3 }, new int[] { 3 });
```

Comment:
This doesn’t need to use the join method.  Just make two int.  Better yet, have the enum produce the int[] for you.

Reply:
LampStatus such as OFF or OTHER have a variety of values of representation.  How do I handle that if the enum are responsible for producing the int[]?

Reply:
Since this code will only be used to generate packets for test, we can just pick one value and use that.

Reply:
As implimented?